lib/extensions/rack_session_dalli_patch.rb

```diff
@@ -4,23 +4,35 @@
 module RackSessionDalliPatch
   def delete_sessions(session_ids)
```

Comment:
Dang, we named this to match the inner method name...We probably _should_ rename this in the future. :/

Reply:
yeah, we can definitely rename the outer to match the inner in a followup.